src/services/openklant2/service.ts

```diff
@@ -471,7 +471,9 @@ export const saveDigitaleAdressen = async (
       verstrektDoorPartij: null,
       adres: adres.adres,
       soortDigitaalAdres:
-        adres.soortDigitaalAdres === "telefoonnummer" ? "telnr" : "email",
+        adres.soortDigitaalAdres === "telefoonnummer"
```

Comment:
Is het niet de bedoeling dat je `soortDigitaalAdres` cast naar `DigitaalAdresTypes`? Dan krijg je `soortDigitaalAdres: adres.soortDigitaalAdres`

Reply:
`digitaleAdressen.push({
      adres: data.emailadres,
      omschrijving: "e-mailadres",
      soortDigitaalAdres: DigitaalAdresTypes.email,
  });`

Reply:
aangepast